.github/workflows/ci.yml

```diff
@@ -7,8 +7,7 @@ on:
       - main
 
 jobs:
-  test:
-    environment: test
+  site-test:
```

Comment:
Can this go back to being called `test`? Then we don't have to reconfigure the repository's "required checks".